src/v2/Apps/Order/Components/SetPayment.tsx

```diff
@@ -41,5 +43,16 @@ export const SetPayment: FC<Props> = props => {
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [])
 
-  return <div>Setting bank debit payment for this order...</div>
+  const SpinnerContainer = styled.div`
+    width: 100%;
+    height: 300px;
+    position: relative;
+  `
+  return (
+    <>
+      <SpinnerContainer>
+        <Spinner position="relative" color="brand" />
+      </SpinnerContainer>
+    </>
```

Comment:
**question/chore**: is there a particular reason why we are wrapping `SpinnerContainer` with `<>` fragment? 🤔